validphys2/src/validphys/coredata.py

```diff
@@ -263,6 +263,10 @@ def additive_errors(self):
         add_table.columns = add_systype["name"].to_numpy()
         return add_table.loc[:, add_table.columns != "SKIP"]
 
+    @property
+    def cuts(self):
+        indices = self.commondata_table.index
+        return slice(indices[0]-1, indices[-1])
```

Comment:
Is this used anywhere?

Reply:
I left a comment at line 2085 in `validphys.simunet_analysis.py` where it is used